pubsub/gcloud/aio/pubsub/subscriber_client.py

```diff
@@ -132,60 +162,56 @@ def create_subscription(self,
             except exceptions.AlreadyExists:
                 pass
 
-        def subscribe(self,
-                      subscription: str,
-                      callback: Callable[[SubscriberMessage], Any],
-                      *,
-                      flow_control: Union[FlowControl, Tuple[int, ...]] = ()
-                      ) -> StreamingPullFuture:
+        async def pull(self, subscription: str, max_messages: int,
```

Comment:
nit: would be awesome to get comments linking to the official docs for each method here, as we do for other gcloud-aio libs. Eg. this one would be:
```python
# https://cloud.google.com/pubsub/docs/reference/rest/v1/projects.subscriptions/pull
async def pull(...)
```